src/ImmersedBoundaries/grid_fitted_immersed_boundaries.jl

```diff
@@ -104,13 +104,13 @@ end
 @inline function ivd_upper_diagonalᵃᵃᶠ(i, j, k, ibg::GFIBG, clock, Δt, νᶜᶜᶜ, κ)
     return ifelse(solid_node(Center(), Center(), Center(), i, j, k+1, ibg),
                   zero(eltype(ibg.grid)),
-                  ivd_upper_diagonalᵃᵃᶜ(i, j, k, ibg.grid, clock, Δt, νᶜᶜᶜ, κ))
+                  ivd_upper_diagonalᵃᵃᶠ(i, j, k, ibg.grid, clock, Δt, νᶜᶜᶜ, κ))
 end
 
 @inline function ivd_lower_diagonalᵃᵃᶠ(i, j, k, ibg::GFIBG, clock, Δt, νᶜᶜᶜ, κ)
     return ifelse(solid_node(Center(), Center(), Center(), i, j, k+1, ibg),
                   zero(eltype(ibg.grid)),
-                  ivd_lower_diagonalᵃᵃᶜ(i, j, k, ibg.grid, clock, Δt, νᶜᶜᶜ, κ))
+                  ivd_lower_diagonalᵃᵃᶠ(i, j, k, ibg.grid, clock, Δt, νᶜᶜᶜ, κ))
```

Comment:
Looks like this function and the one above are identical?

Reply:
the one above is the upper diagonal and this one is the lower? Or you mean the change?

Reply:
Ahh, I was focused on the three letter code and missed that one was `upper` and one was `lower`.